### DISCUSSION TITLE
Can we set the OIDC redirect URI request content-type?

### DISCUSSION THREAD

**Discussion by AnirudhPokala** [OpenIDC/mod_auth_openidc#819] [Q&A] [answered]
I have set an OIDCRedirectURI in apache configuration to support keystone with OIDC and after successful google login it does a POST call on redirect URI  with id_token the Content-Type being used is `application/x-www-form-urlencoded`. But the open-stack keystone is throwing error with `"Expecting to find application/json in Content-Type header. The server could not comply with the request since it is either malformed or otherwise incorrect. The client is assumed to be in error."` 
Is there any configuration parameter to set the content-type for OIDC from conf file?

**Comment by zandbelt** [comment, marked as answer]
no, there is not and there shouldn't be; the content type is standards compliant; I think you'll need to review Keystone docs to fix this because there's samples out there that show how to configure this (most probably without setting OIDCResponseType to form_post)

**Reply by AnirudhPokala** [reply]
Sure, will check and also try by removing the form_post type. Thanks for the reply @zandbelt
